code/__DEFINES/mobs.dm

```diff
@@ -837,35 +837,37 @@ GLOBAL_LIST_INIT(layers_to_offset, list(
 #define HEAL_CLONE (1<<5)
 /// Heals all stamina damage.
 #define HEAL_STAM (1<<6)
+/// Heals all nutrition damage.
+#define HEAL_NUTR (1<<7)
```

Comment:
why changing everything here instead of just going (1<<20)?

Reply:
This is to ensure that nutritional damage loads with the rest of the damages so it doesn't cause any issues